.github/workflows/php-test.yml

```
@@ -0,0 +1,63 @@
+name: Unit Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
+      - 'release/**'
+    # Only run if PHP-related files changed.
+    paths:
+      - '.github/workflows/php-lint.yml'
+      - '**.php'
+      - 'phpcs.xml.dist'
+      - 'phpstan.neon.dist'
+      - 'composer.json'
+      - 'composer.lock'
+  pull_request:
+    # Only run if PHP-related files changed.
+    paths:
+      - '.github/workflows/php-lint.yml'
+      - '**.php'
+      - 'phpcs.xml.dist'
+      - 'phpstan.neon.dist'
+      - 'composer.json'
+      - 'composer.lock'
```

Some file references need to be replaced based on this being about PHPUnit, plus the correct GitHub workflow file:
```suggestion
    paths:
      - '.github/workflows/php-test.yml'
      - '**.php'
      - 'phpunit.xml.dist'
      - 'composer.json'
      - 'composer.lock'
  pull_request:
    # Only run if PHP-related files changed.
    paths:
      - '.github/workflows/php-test.yml'
      - '**.php'
      - 'phpunit.xml.dist'
      - 'composer.json'
      - 'composer.lock'
```

---

Oops! Good catch!